logging/src/commonMain/kotlin/log.kt

```diff
@@ -0,0 +1,267 @@
+package com.splendo.mpp.log
+
+import ru.pocketbyte.hydra.log.HydraLog
+
+/**
+ * Initializes HydraLog with given logger.
+ *
+ * Be aware that once initialized HydraLog cannot be initialised with new loggers. You will get first logger you have used for initialisation.
+ *
+ * @param logger - logged to be used for log output
+ * @return first logger used for initialisation.
+ */
+fun initLogger(logger: Logger): Logger {
+    try {
+        HydraLog.init(InternalLogger(logger))
+    } catch (e: IllegalStateException) {
+        println("LOG: HydraLog was already initialised. {${e.message}}")
+    }
+
+    return logger()
+}
+
+/**
+ * @return first logger you have used for initialisation.
+ */
+fun logger(): Logger {
+    return (HydraLog.logger as InternalLogger).logger
```

Comment:
We should catch the cast exception and wrap it in `IllegalStateException` or something similar. It means someone initialized Hydralog before us, it's good to inform the user about this rather than crash. It could happen e.g. if someone already uses Hydralog for a project.